mcenter_cli/parallelm/mcenter_cli/command_action_log.py

```diff
@@ -0,0 +1,16 @@
+from cliff.command import Command
+
+from parallelm.mcenter_cli.list_actions import list_actions
+
+
+class ActionLogListCommand(Command):
+    def get_parser(self, prog_name):
+        parser = super(ActionLogListCommand, self).get_parser(prog_name)
+        parser.add_argument('--object', action='store', help='List of Objects for which to get actions')
+        parser.add_argument('--action', action='store', help='List of actions to query')
```

Comment:
"actions" should be "Actions" to follow the pattern. "List of Objects", for example. Capitalize the word is my point, or done capitalize, but be consistent.

Reply:
fixed